Create public app view

JuanequeX opened the pull request:
# Description
 - Create the public view
 - This view have two options for employee user: Check in and Check out
 - This view add a button to login and logout admin.
 - Add a clock that gives info about current hour and day
 
# What?
- I've added support for employee authentication with private a key to make check in or check out

# Why?
- These changes benefit the admin login and the services that produce interation with the employees

  
  ## Screenshots 
![desktop-clock](https://user-images.githubusercontent.com/50384228/136289337-dc70c3f7-2a99-4490-a98d-1ace7ca64a0d.png)
![mobile-clock](https://user-images.githubusercontent.com/50384228/136289342-1201aa7b-22eb-4c57-b423-8eddb6ec9f75.png)

jclopezdev commented on app/views/layouts/application.html.erb:
```diff
@@ -7,16 +7,17 @@
     <%= csp_meta_tag %>
     <%= stylesheet_pack_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <script src="https://kit.fontawesome.com/e03d7ee539.js" ></script>
```
maybe I don't understand this very well, but is this the best way to add `fontawesome` to the project?

JuanequeX replied:
We don't focus to add icons manually svg because we want to have advantage to reduce the band width, what do you think exist another better option that you want share with us?

jclopezdev commented on .rubocop.yml:
```diff
@@ -1,6 +1,6 @@
 AllCops:
   NewCops: enable
-  TargetRubyVersion: 2.5.1
+  TargetRubyVersion: 3.0.0
```
I'd add a different commit for this change since it doesn't have anything to do with this PR, what do you think?

YilKanda reviewed (commented):
Why was the `PULL_REQUEST_TEMPLATE.md` file deleted?

JuanequeX commented:
@YilKanda  I deleted because, we aggregated it before that Carlos will add the new template in the project, we had two templates in the project the first that we added and the another one that added Carlos, we left only the one that Carlos placed
